### PR TITLE
Workaround to avoid history bug #1324

### PR DESCRIPTION
**Pull request by Jojo144**
This is a poor workaround to bug #1324.
It does not solve the issue but at least it avoid the "Internal server error", which is nice.

Now the bills for which payer or ower are not well versioned seem to:
image

If anyone has the same issue as me, I found a trick to make members well versioned: deactivate and reactivate them. The bills recorded afterward are well versioned.

### CODE REVIEW

File: ihatemoney/history.py [hunk 1]
```diff
@@ -38,7 +38,10 @@ def history_sort_key(history_item_dict):
 
 def describe_version(version_obj):
     """Use the base model str() function to describe a version object"""
-    return parent_class(type(version_obj)).__str__(version_obj)
+    if version_obj is None:
```
Comment (almet) [inline ihatemoney/history.py:41]:
```suggestion
    if not version_obj:
```

**Comment by almet** [comment]
Thanks :+1: Added a suggestion, other than that it seems fine by me

**Review by almet — APPROVED** [review]

File: ihatemoney/history.py (hunk 1 above)
Comment (Jojo144) [reply ihatemoney/history.py:41]: It's a few characters shorter but less clear I feel. Anyway, I don't mind a lot...

Comment (almet) [reply ihatemoney/history.py]: Let me explain why I added this comment (sorry if this felt a bit short). This is not testing the exact same thing, so it's not only a matter of preference.

Given the fact that we're not using types in the project (we could), testing for Falsey values allows to consider both `None` and `False`. This is a general thing I try to do in these situations. Here it doesn't really matter to be honest, but that's my natural tendency.

Feel free to update your code or not, and we can merge :-)

Comment (Jojo144) [reply ihatemoney/history.py]: Interesting, thanks.

**Pull request merged**